README.md

```diff
@@ -19,23 +18,17 @@ keys/key containers.
 To install globally:
 
 ```powershell
-Install-Module -Name 'Carbon.Permissions'
-Import-Module -Name 'Carbon.Permissions'
+Install-Module -Name 'Carbon.Security'
+Import-Module -Name 'Carbon.Security'
 ```
 
 To install privately:
 
 ```powershell
-Save-Module -Name 'Carbon.Permissions' -Path '.'
-Import-Module -Name '.\Carbon.Permissions'
+Save-Module -Name 'Carbon.Security' -Path '.'
+Import-Module -Name '.\Carbon.Security'
 ```
 
 ## Commands
 
 * `Get-CAcl`: Gets the access control (i.e. security descriptor) for a file, directory, or registry key.
-* `Get-CPermission`: Gets the permissions (access control rules) for a file, directory, registry key, or certificate
-  private key/key container.
-* `Grant-CPermission`: Grants permissions on a file, directory, registry key, or certificate private key/key container.
-* `Revoke-CPermission`: Revokes permissions on a file, directory, registry key, or certificate private key/key
-  container.
-* `Test-CPermission`: Tests permissions on a file, directory, registry key, or certificate private key/key container.

```

Comment:
Did you mean to remove these? These functions still exist in the module.

Reply:
Yes. But they are considered internal functions and shouldn't be used by regular folk, only by other Carbon modules. If people find them, fine, but they should be using `Carbon.FileSystem`, `Carbon.Registry`, and `Carbon.Cryptography` instead. The docs on each function explain that.

Reply:
Ah ok, good to know. I saw them exported in the `.psd1` and expected them to be public.